app/main.py

```diff
@@ -1,214 +1,73 @@
+from typing import Dict
+from app.knight import Knight
+from app.battle import Battle
+
 KNIGHTS = {
     "lancelot": {
         "name": "Lancelot",
-        "power": 35,
-        "hp": 100,
-        "armour": [],
-        "weapon": {
-            "name": "Metal Sword",
-            "power": 50,
-        },
+        "power": 50,
+        "hp": 80,
+        "armour": [{"part": "helmet", "protection": 10}],
+        "weapon": {"name": "Sword", "power": 30},
         "potion": None,
     },
     "arthur": {
         "name": "Arthur",
         "power": 45,
         "hp": 75,
         "armour": [
-            {
-                "part": "helmet",
-                "protection": 15,
-            },
-            {
-                "part": "breastplate",
-                "protection": 20,
-            },
-            {
-                "part": "boots",
-                "protection": 10,
-            }
+            {"part": "helmet", "protection": 15},
+            {"part": "breastplate", "protection": 20},
+            {"part": "boots", "protection": 5}
         ],
-        "weapon": {
-            "name": "Two-handed Sword",
-            "power": 55,
-        },
-        "potion": None,
+        "weapon": {"name": "Sword", "power": 25},
+        "potion": {"name": "Blessing", "effect": {"hp": 20, "power": 10}},
     },
     "mordred": {
         "name": "Mordred",
-        "power": 30,
-        "hp": 90,
-        "armour": [
-            {
-                "part": "breastplate",
-                "protection": 15,
-            },
-            {
-                "part": "boots",
-                "protection": 10,
-            }
-        ],
-        "weapon": {
-            "name": "Poisoned Sword",
-            "power": 60,
-        },
-        "potion": {
-            "name": "Berserk",
-            "effect": {
-                "power": +15,
-                "hp": -5,
-                "protection": +10,
-            }
-        }
+        "power": 55,
+        "hp": 85,
+        "armour": [{"part": "shield", "protection": 30}],
+        "weapon": {"name": "Axe", "power": 35},
+        "potion": None,
     },
     "red_knight": {
         "name": "Red Knight",
         "power": 40,
         "hp": 70,
-        "armour": [
-            {
-                "part": "breastplate",
-                "protection": 25,
-            }
-        ],
-        "weapon": {
-            "name": "Sword",
-            "power": 45
-        },
-        "potion": {
-            "name": "Blessing",
-            "effect": {
-                "hp": +10,
-                "power": +5,
-            }
-        }
-    }
+        "armour": [{"part": "breastplate", "protection": 25}],
+        "weapon": {"name": "Sword", "power": 45},
+        "potion": {"name": "Blessing", "effect": {"hp": 10, "power": 5}},
+    },
 }
```

Comment:
Let's move config to another file like `config.py ` (and do not change data, please)